### PR TITLE
New tokens/alpha bravo

### PR DESCRIPTION
**Pull request by therealMikkoK**
second batch of the week, here we go.

### CODE REVIEW

File: tokens/0x00c4b398500645eb5da00a1a379a88b11683ba01.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- Blog: https://medium.com/eximchain
+- CoinGecko: https://www.coingecko.com/en/coins/eximchain/
+- CoinMarketCap: https://coinmarketcap.com/currencies/eximchain/
+- Email: mailto:hello@eximchain.com
+- Facebook: https://www.facebook.com/eximchain/
+- Github: https://github.com/eximchain
+- Telegram: https://t.me/eximchain
+- Twitter: https://twitter.com/eximchainexc
+- Website: https://eximchain.com
+- Website: https://www.eximchain.com
```
Comment (freeatnet) [inline tokens/0x00c4b398500645eb5da00a1a379a88b11683ba01.yaml]: Duplicate website entry. (Differs by a `www.`)

File: tokens/0x00c4b398500645eb5da00a1a379a88b11683ba01.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Whitepaper: http://eximchain.com.s3.amazonaws.com/Whitepaper%20-%20Eximchain.pdf
```
Comment (freeatnet) [inline tokens/0x00c4b398500645eb5da00a1a379a88b11683ba01.yaml]: Website has https://www.eximchain.com/documents/Whitepaper-Eximchain.pdf for a whitepaper link. We could consider using that instead.

File: tokens/0xf19eac69f9566f0b40660bd0cc16234330f38456.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  CraftR, a global decentralized P2P marketplace for buying and selling digital creative
+  assets through the blockchain.
+links:
+- Bitcointalk: https://bitcointalk.org/index.php?topic=3053063
+- Blog: https://medium.com/@craftrdev
+- Email: mailto:info@craftr.io
+- Facebook: https://www.facebook.com/CraftrToken
+- Github: https://github.com/CraftrToken
+- Linkedin: https://www.linkedin.com/company/craftrproject/
+- Reddit: https://www.reddit.com/r/CryptalDashCommunity/
```
Comment (freeatnet) [inline tokens/0xf19eac69f9566f0b40660bd0cc16234330f38456.yaml]: > Official Subreddit for CryptalDash Exchange www.cryptaldash.com

I don't think that's the right subreddit.

File: tokens/0xf4faea455575354d2699bc209b0a65ca99f69982.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  With the No BS Token the community reviews ICOs, Exchanges, and other projects to separate the BS from the No BS.
+links:
+- Bitcointalk: https://bitcointalk.org/index.php?topic=4709590
+- CoinGecko: https://www.coingecko.com/en/coins/no-bs-crypto/
+- CoinMarketCap: https://coinmarketcap.com/currencies/no-bs-crypto/
+- Facebook: https://www.facebook.com/groups/nobscrypto/
+- Telegram: https://t.me/joinchat/Gn2Ocg_ugvLie-2VD7w4ig
+- Twitter: https://twitter.com/nobscrypto
+- Website: https://nobscrypto.com
+- Website: https://nobscrypto.com/
```
Comment (freeatnet) [inline tokens/0xf4faea455575354d2699bc209b0a65ca99f69982.yaml]: Duplicate website, _sigh_.

Comment (therealMikkoK) [reply tokens/0xf4faea455575354d2699bc209b0a65ca99f69982.yaml]: there were plenty of those this run.

File: tokens/0x00c4b398500645eb5da00a1a379a88b11683ba01.yaml (hunk 1 above)
Comment (therealMikkoK) [reply tokens/0x00c4b398500645eb5da00a1a379a88b11683ba01.yaml]: deleted extra link

File: tokens/0x00c4b398500645eb5da00a1a379a88b11683ba01.yaml (hunk 2 above)
Comment (therealMikkoK) [reply tokens/0x00c4b398500645eb5da00a1a379a88b11683ba01.yaml]: changed link to HTTPS

**Comment by therealMikkoK** [comment]
@freeatnet you want to do this on Monday?

**Pull request merged**